dev-requirements.txt

```diff
@@ -1,4 +1,5 @@
-elastic-transport>=8.0.0b1, <9
+# TODO switch back to elastic-transport>=8,<9 between elastic-transport release and elasticsearch-py release
+elastic-transport @ git+https://github.com/elastic/elastic-transport-python
```

Comment:
I vaguely remember some pain in specifying git urls without an hash, like when trying to upgrade a package checked out from git.
BTW you didn't release a elastic-transport-python because you want to check that all layers are working right?

Reply:
That's right. I don't intend to release anything with git dependencies. Maybe I should use a pre-release instead though!